.gitignore

```diff
@@ -22,3 +22,7 @@ server/configs
 docs/build
 public/js
 dist
+
+#Vagrant
```

Comment:
what brought about these changes to the .gitignore?

Reply:
This was removed, found a better way to test the clients environment. 
[8b188e4](https://github.com/AMP-SCZ/dpdash/pull/75/commits/8b188e4dd3522e4ca13f10b2083b5a0224dfaabf)